frontend/src/main/scala/akkaviz/frontend/ActorRepository.scala

```diff
@@ -0,0 +1,47 @@
+package akkaviz.frontend
+
+import rx.Var
+
+import scala.scalajs.js
+import scala.scalajs.js.Date
+
+class ActorRepository {
+
+  import ActorRepository._
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/lustefaniak/akka-viz/file/2125674775/issues/source?bid=3121282&fileBranchId=3147453#l10)